nile-backend/src/main/java/com/tp/Nile/controllers/ReviewController.java

```diff
@@ -15,7 +15,7 @@
     @Autowired
     ReviewServiceImpl service;
 
-    @PostMapping
+    @PostMapping("/addReview")
```

Comment:
Please just leave this as @PostMapping and I'm going to say the same thing for every other route where it's unnecessary.